### DISCUSSION TITLE
Which package should I use for Macintosh(real MAC).  Not Hackintosh

### DISCUSSION THREAD

**Discussion by lovelytan** [chris1111/Wireless-USB-Big-Sur-Adapter#214] [Q&A] [answered]
I want to use dual NICs on my MacBook Pro ，Which package should I use， OC or Clover, v13 or others?

For Macintosh (real MAC), NOT Hackintosh.

Thanks!

**Comment by chris1111** [comment, marked as answer]
This one [Wireless-USB-Big-Sur-Adapter](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter)
